Joplin_install_and_update.sh

```diff
@@ -16,11 +16,14 @@ handleError() {
 #-----------------------------------------------------
 # Variables
 #-----------------------------------------------------
-COLOR_RED=`tput setaf 1`
-COLOR_GREEN=`tput setaf 2`
-COLOR_YELLOW=`tput setaf 3`
-COLOR_BLUE=`tput setaf 4`
-COLOR_RESET=`tput sgr0`
+# tput command will fail if $TERM is set to dumb (ex : ansible deployment)
+if [[ "${TERM}" != "dumb" ]] ; then
```

Comment:
```suggestion
if [[ $TERM != "dumb" ]]; then
```

This is a valid use case. However, I'm wondering if someone wouldn't rather write the script logic as ansible code instead.

But I'm ok with merging this as well.

Reply:
I ran a few tests. Can you check your ansible workstation? It seems that it sends the `TERM=dumb`.

I ran a few tests, because I remembered that ansible might set TERM to the local TERM.

Create a file `/tmp/myt.sh` on the remote machine:

```bash
#!/bin/bash

echo "Shell: $TERM"

COLOR_RED=`tput setaf 1`
COLOR_GREEN=`tput setaf 2`
COLOR_YELLOW=`tput setaf 3`
COLOR_BLUE=`tput setaf 4`
COLOR_RESET=`tput sgr0`

echo "${COLOR_GREEN}Here should be colored text.${COLOR_RESET}"
```

Then run from your ansible box:

```
ansible <remote-hostname> -m shell -a '/tmp/myt.sh'
```

When I run this, I get the following:

<img width="231" alt="image" src="https://github.com/laurent22/joplin/assets/223439/613e3d6c-ed25-481b-8afb-015385de5d04">

However, when I do this

```
export TERM=bla
ansible <remote-hostname> -m shell -a '/tmp/myt.sh'
```

the following happens:

<img width="472" alt="image" src="https://github.com/laurent22/joplin/assets/223439/46e89a57-4b0e-4c36-aed3-9d1aac7c0c1e">

Reply:
Rather than checking for one specific terminal, can't we check that the command `tput` is available? That way it will more generally work with all terminals

Reply:
Yes and no. Checking for `tput` is not a bad idea anyway, but this is an issue where tput errors out, because the terminal in not recognized. So we could additionally check for a return code of the tput command.

However, I am not so sure anymore, whether we should do this at all. As I explained earlier, I believe something's off with the OP's system. My first reply was a bit too hasty. Shortly after a clicked the button, I recalled how ansible actually handles TERM vars.

I would suggest to close this PR and I'll fix it differently.

Reply:
I'm fine going with your recommendation since I don't know much about this. Feel free to close and reopen with a different change if that makes sense to you

Reply:
Ok, I'm closing the PR. I'll push the fix directly since I recall that I am allowed to merge PRs to this script anyway.